### PR TITLE
style: Use panels for issue lists

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/emptyStateWarning.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'react-emotion';
+// import {Panel, PanelBody} from './panels';
+
+class EmptyStateWarning extends React.Component {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/emptyStateWarning.jsx]: Can we extend `<EmptyMessage>` here?

File: src/sentry/static/sentry/app/components/issueList.jsx [hunk 2]
```diff
@@ -96,20 +97,24 @@ const IssueList = createReactClass({
...
-                orgId={params.orgId}
-                statsPeriod={this.props.statsPeriod}
-                showActions={this.props.showActions}
-              />
-            );
-          })}
-        </ul>
+        <Panel>
+          <PanelBody>
+            <div className="issue-list">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/issueList.jsx]: Can `className` be moved onto `PanelBody`?

**Review by billyvg — COMMENTED** [review]
Empty state text looks like it should be centered in org dashboard.

Do we have a list of views/urls where these components are used? I'd like to add some percy snapshots for them since they probably aren't manually tested much.

File: src/sentry/static/sentry/app/components/emptyStateWarning.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/components/emptyStateWarning.jsx]: Apart from their purpose there's not a lot that's actually the same, would rather keep 2 simple components than make one complicated one

**Comment by lynnagara** [comment]
I don't have a list of views

File: src/sentry/static/sentry/app/components/emptyStateWarning.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/components/emptyStateWarning.jsx]: It might be worthwhile to eventually align the design down the line though

**Review by billyvg — COMMENTED** [review]
There's nested Panels on the Releases/Overview page

image

**Comment by billyvg** [comment]
These views also have an extra border:

image

(Assigned to me, bookmarks, recently viewed)

**Review by billyvg — APPROVED** [review]

**Pull request merged**